src/Raven.Studio/typescript/models/resources/info/databaseInfo.ts

```diff
@@ -209,6 +212,8 @@ class databaseInfo {
             
             this.nodes(joinedNodes);
         }
+
+        this.databaseAccessText(accessManager.default.getDatabaseAccessLevelText(this.name));
```

Comment:
this is obervable but it won't be updated when security rule will be changed, so either make this plain string or convert to computed.

Reply:
ok as is (in update method)